client/html/templates/account/profile/account-body-standard.php

```diff
@@ -0,0 +1,190 @@
+ * that create parts of the output displayed in the generated HTML page. Controller
+ * names are usually alpha-numeric.
+ *
+ * @param string Name of the controller
+ * @since 2019.10
+ * @category Developer
+ * @see client/html/account/profile/url/target
+ * @see client/html/account/profile/url/action
+ * @see client/html/account/profile/url/config
+ */
+
+/** client/html/account/profile/url/action
+ * Name of the action that should create the output
+ *
+ * In Model-View-Controller (MVC) applications, actions are the methods of a
+ * controller that create parts of the output displayed in the generated HTML page.
+ * Action names are usually alpha-numeric.
+ *
+ * @param string Name of the action
+ * @since 2019.10
+ * @category Developer
+ * @see client/html/account/profile/url/target
+ * @see client/html/account/profile/url/controller
+ * @see client/html/account/profile/url/config
+ */
+
+/** client/html/account/profile/url/config
+ * Associative list of configuration options used for generating the URL
+ *
+ * You can specify additional options as key/value pairs used when generating
+ * the URLs, like
+ *
+ *  client/html/<clientname>/url/config = array( 'absoluteUri' => true )
+ *
+ * The available key/value pairs depend on the application that embeds the e-commerce
+ * framework. This is because the infrastructure of the application is used for
+ * generating the URLs. The full list of available config options is referenced
+ * in the "see also" section of this page.
+ *
+ * @param string Associative list of configuration options
+ * @since 2019.10
+ * @category Developer
+ * @see client/html/account/profile/url/target
+ * @see client/html/account/profile/url/controller
+ * @see client/html/account/profile/url/action
+ * @see client/html/url/config
+ */
+
+/** client/html/account/profile/url/filter
+ * Removes parameters for the detail page before generating the URL
+ *
+ * For SEO, it's nice to have URLs which contains only required parameters.
+ * This setting removes the listed parameters from the URLs. Keep care to
+ * remove no required parameters!
+ *
+ * @param array List of parameter names to remove
+ * @since 2019.10
+ * @category User
+ * @category Developer
+ * @see client/html/account/profile/url/target
+ * @see client/html/account/profile/url/controller
+ * @see client/html/account/profile/url/action
+ * @see client/html/account/profile/url/config
+ */
+
+$passwordErrors = $this->get('passwordErrors', []);
+
+?>
+<?php $this->block()->start( 'account/profile/account' ) ?>
+
+<div class="account-profile-account">
+    <h1 class="header"><?= $enc->html( $this->translate( 'client', 'account' ) ) ?></h1>
+    <form class="container-fluid" method="POST" action="<?= $enc->attr( $this->link( 'client/html/account/profile/url' ) ) ?>">
+        <?= $this->csrf()->formfield() ?>
+        <?php if ( $this->get('passwordChanged', '') === 'true' ) : ?>
+            <div class="row d-flex justify-content-center">
```

Comment:
Utility CSS classes are handy but a bad idea if different themes have to cope with them. Please add the necessary styles to the CSS file.

Reply:
changed that